tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -854,16 +855,143 @@ impl<'a, F> fmt::Debug for FieldFnVisitor<'a, F> {
     }
 }
 
+// === printing synthetic Span events ===
+
+/// Select the degree to which tracing spans are logged as events
```

Comment:
How about 
```suggestion
/// Configures what points in the span lifecycle are logged as events.
```
and maybe link to the docs for `with_spans`?